shared/teams/team/index.js

```diff
@@ -409,8 +416,18 @@ class Team extends React.PureComponent<Props> {
                   </Text>
                 </Box>
               </Box>
+
+              <Box style={{...stylesSettingsTabRow, alignSelf: 'center', paddingTop: globalMargins.xtiny}}>
```

Comment:
I believe this should be outside the `{admin && ` block, I can't save my settings on teams I'm not an admin of

Reply:
Great catch, thanks! Fixed.